packages/block-library/src/video/edit.js

```diff
@@ -245,6 +247,11 @@ class VideoEdit extends Component {
 								setAttributes( { caption: value } )
 							}
 							inlineToolbar
+							__unstableOnSplitAtEnd={ () =>
+								insertBlocksAfter(
+									createBlock( 'core/paragraph' )
+								)
+							}
```

Comment:
I don't think the changes in this file have any effect on mobile because mobile uses the `BlockCaption` component in `video/edit.native.js`. You're probably aware of this, but on mobile we have a separate reusable caption component, whereas on web the captions were being created separately when needed (although it sounds like that may be changing).

Reply:
The idea was to implement the caption break on video web, but it looks https://github.com/WordPress/gutenberg/pull/22934 addressed that. So this will go away!